manila/share/drivers/netapp/dataontap/cluster_mode/lib_multi_svm.py

```diff
@@ -611,17 +622,18 @@ def _delete_vserver_without_lock():
             self._client.delete_vserver(vserver,
                                         vserver_client,
                                         security_services=security_services)
-            ipspace_deleted = False
-            if (ipspace_name and ipspace_name not in CLUSTER_IPSPACES
-                    and not self._client.ipspace_has_data_vservers(
-                        ipspace_name)):
-                self._client.delete_ipspace(ipspace_name)
-                ipspace_deleted = True
-
-            if not ipspace_name or ipspace_deleted:
-                # NOTE(dviroel): only delete vlans if they are not being used
-                # by any ipspaces and data vservers.
-                self._delete_vserver_vlans(interfaces_on_vlans)
+
+            ipspace_deleted = self._client.delete_ipspace(ipspace_name)
+
+            # NOTE(dviroel): only delete vlans if they are not being used
+            # by any ipspaces and data vservers.
+            if ipspace_name and not ipspace_deleted:
+                # NOTE(carthaca): filter for degraded ports only, those where
```

Comment:
We did a review session with Chuan: we don't just want the degraded ports here: also the vlan ports belonging to the vserver that we are currently deleting should be added (they may not be marked as degraded, yet, depending how fast netapp monitoring would notice)